routes/api/event.js

```diff
@@ -2,32 +2,41 @@ const express = require('express');
 const router = express.Router();
 const eventModel = require('../../models/eventSchema.js')
 const mongoose = require('mongoose');
+const {getToken, isUser, isAdmin} = require('../auth.js')
 
-router.post('/event', function(req, res, next) {
-  const event = new eventModel({
+router.post('/event', isAdmin, async function(req, res) {
+  const event = {
     title: req.body.title,
     description: req.body.description,
     place: req.body.place,
     fromDate: req.body.fromDate,
     toDate: req.body.toDate
-  });
-  event.save((err, result) => {
-    if (err) {
-      console.log('error ' + err);
-      return next(err);
-    }
-    res.send({'message':'event added successfully'})
-  })
+  }
+  await eventModel.create(event)
+  res.send({message:'event added successfully'})
 });
 
-/*router.get('/users', function(req, res,next) {
-  //var User = mongoose.model('User',userModel);
-  var query = userModel.find({},{username:1,_id:0});
-  query.exec(function(err,usernames){
-    if (err)
-      return next(err);
-    res.send(usernames);
-    });
-});*/
+router.get('/events', isUser, async function(req, res) {
+  const events = await eventModel.find({});
```

Comment:
Do we want to send all events in the payload?

Reply:
For now, need to discuss with @jenisonleo, may be changed to pagination kind of functionality.